test/e2e/common/node/configmap.go

```diff
@@ -258,6 +306,21 @@ func newConfigMap(f *framework.Framework, name string) *v1.ConfigMap {
 	}
 }
 
+func newSpecialKeyConfigMap(f *framework.Framework, name string) *v1.ConfigMap {
```

Comment:
Can you comment this - what makes it special?  You could also just do this in-line - it's only used one time.

Please comment it